src/Modules/LearningTasks/Tutor.LearningTasks.Core/Domain/LearningTasks/LearningTask.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Tutor.BuildingBlocks.Core.Domain;
+
+namespace Tutor.LearningTasks.Core.Domain.LearningTasks;
+
+public class LearningTask : AggregateRoot
+{
+    public int UnitId { get; private set; }
+    public string? Name { get; private set; }
+    public string? Description { get; private set; }
+    public bool IsTemplate { get; private set; }
+    public DomainModel? DomainModel { get; private set; }
+    public List<CaseStudy>? CaseStudies { get; private set; }
+    public List<Step> Steps { get; private set; }
+    public double MaxPoints {  get; private set; }
+
+    public void Update(LearningTask learningTask)
+    {
+        UnitId = learningTask.UnitId;
+        Name = learningTask.Name;
+        Description = learningTask.Description;
+        IsTemplate = learningTask.IsTemplate;
+        DomainModel = learningTask.DomainModel;
+        CaseStudies = learningTask.CaseStudies;
+        UpdateSteps(learningTask.Steps);
+        MaxPoints = Steps.Sum(s => s.MaxPoints);
+    }
+
+    private void UpdateSteps(List<Step> steps)
+    {
+        foreach (var step in steps)
```

Comment:
For every step I need to calculate `maxPoints` which is done inside `existingStep?.Update(step);`. In this moment `Steps=steps;` will work because of this:
```
CreateMap<LearningTaskDto, LearningTask>()
            .AfterMap((src, dest) => dest.SetMaxPoints()).ReverseMap();
```
Another solution is without calculate method for example I can expect that I will get dto where everything is calculated correct and remove method for updating maxPoints. For example, I can calculate step points and learning tasks points in frontend app.